searches/search_quadruplet.py

```diff
@@ -0,0 +1,72 @@
+def search_quadruplets(sequence: list[int], target: int) -> list[int]:
+    """
+    >>> search_quadruplets([4, 1, 2, -1, 1, -3], 1)
+
+    [-3, -1, 1, 4], [-3, 1, 1, 2]
+    Explanation: Both the quadruplets add up to the target.
+
+    >>> search_quadruplets([2, 0, -1, 1, -2, 2], 2)
+    [-2, 0, 2, 2], [-1, 0, 1, 2]
+    Explanation: Both the quadruplets add up to the target.
+
+    Technique: Two Pointers
+    This problem follows the Two Pointers pattern and
+    shares similarities with Triplet Sum to Zero.
+
+    """
```

Comment:
```suggestion
    """
    [Explain the problem here]
    Technique: Two Pointers
    This problem follows the Two Pointers pattern and
    shares similarities with Triplet Sum to Zero.

    >>> search_quadruplets([4, 1, 2, -1, 1, -3], 1)
    [-3, -1, 1, 4], [-3, 1, 1, 2]
    >>> search_quadruplets([2, 0, -1, 1, -2, 2], 2)
    [-2, 0, 2, 2], [-1, 0, 1, 2]
    """
```
Please reformat the docstring and explain the problem.